ros2_control_demo_bringup/config/rrbot_joint_trajectory_publisher.yaml

```diff
@@ -1,7 +1,7 @@
 publisher_joint_trajectory_controller:
   ros__parameters:
 
-    controller_name: "position_trajectory_controller"
+    controller_name: "joint_trajectory_controller"
```

Comment:
I will change the name of this controller to joint_trajectory_position_controller like in #190

Reply:
This file has been deleted and the gazebo example also uses the rrbot_controllers.yaml